modules/ROOT/pages/osv2-tutorial.adoc

```diff
@@ -3,263 +3,388 @@ ifndef::env-site,env-github[]
++
+.The arrow shows the *Create new application* option.
+image::os-tutorial-dc-create-app.png[Create new application option]
++
+. In the *New Mule Application* window, enter `osv2demo` for the project name and click *Create*.
+. In the wizard, click *Go straight to canvas*:
 +
-image::os-dc-http-trigger.png[]
+.The arrow shows the *Go straight to canvas* option.
+image::os-tutorial-dc-goto-canvas.png[Go straight to canvas]
 +
-. Specify the *Path* as `/store`.
-. Click *Edit*:
+. Click the *Trigger* box and enter `H` to search for the HTTP Listener:
 +
-image::os-dc-http-edit.png[]
+.The arrow shows the HTTP Listener.  
+image::os-tutorial-dc-http-search.png[Search for the HTTP Listener]
 +
-. Click *Save*. You can use the default settings.
+. Click *HTTP Listener*.
+. In the HTTP Listener window, set the value of *Path* to `/store` and then click the *Close* button (X):
 +
-image::os-dc-http-settings.png[]
+.The screenshot shows (*1*) the *Path* value and (*2*) the *Close* button (X).
+image::os-tutorial-dc-http-path.png[Path value and Close button]
 +
-. Click the plus sign to the right of the trigger:
+Because the default settings for HTTP Listener are correct, you don't need to click *Edit*.
+. Click the plus icon (+) to the right of the HTTP Listener trigger:
 +
-image::os-dc-plus-sign.png[]
+.The arrow shows the plus icon (+).
+image::os-tutorial-dc-plus-icon.png[Plus icon]
 +
-. In the *Select a component* window, search for *Obj*, then click *ObjectStore Connector*,
-and click the *Store* operation.
+. In the *Select a component* window, enter `ob` to search for the ObjectStore Connector:
 +
-image::os-dc-os-store-op.png[]
+.The arrow shows the ObjectStore Connector.
+image::os-tutorial-dc-os-search.png[Search for the Object Store Connector]
 +
-. Specify the *Key* as `#[payload.key]` and the *Value* with the `payload.value` string.
+. Click the connector, and then click the *Store* operation:
 +
-image::os-dc-store-setup.png[]
+.The arrow shows the *Store* operation.
+image::os-tutorial-dc-store-select.png[Store operation]
+. In the *Configuration* tab, specify `#[payload.key]` in the *Key* field and `payload.value` in the *Value* field:
 +
-. Click the *Run* button at the top of the Design Center window. When successful, *Running* appears.
+.The screenshot shows (*1*) the *Key* field and (*2*) the *Value* field.
+image::os-tutorial-dc-store-values.png[Key and Value fields for the Store operation]
+. Click *Test* to run the app in a temporary location.
 +
-image::os-dc-run-button.png[]
+The app starts running in a temporary environment:
 +
-. To the right of the *Run* button are three vertical dots. Click the dots, then click *Copy link*.
+.The arrow shows a message indicating the `osv2demo` app is running in a temporary environment.
+image::os-tutorial-dc-running.png[The osv2demo app running in a temporary environment]
+
+You have successfully used Design Center to configure and deploy an app with an object store.
```

Comment:
...but if the app is running in a temp environment, is it really deployed?

Reply:
Not too keen on saying "you have been successful" when we basically assume that our tasks work. 
I can see this for Citizen docs, but maybe not others.
How about "After you use Design Center to configure and deploy..., you can test the connection by...."

Reply:
Also, if the culmination of this topic set is configuring and deploying, then the container topic should probably be Configure and Deploy rather than Create and Configure.

Reply:
See slack (and the MQ Tutorial).